Task_30.2

```diff
@@ -0,0 +1,6 @@
+select first_name as 'First Name', last_name as 'Last Name' from employees;

```

Comment:
Please keep the habit of writing the keywords in uppercase. This may be important in some environments (for example, to search and distinguish SQL queries from other code by uppercase in a large project)

Reply:
ok, thanks!